CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.28.0] - 2025-03-06
```

Comment:
It might not seem necessary to be a major release, but `Runner` now becomes `!UnwindSafe`. 🥵